### PR TITLE
Feature Set .gitattributes to ignore docs and tests folder, and …

### PR DESCRIPTION
**Pull request by flamerohr**
some source asset files which are not needed

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -2,6 +2,10 @@ language: php
 
 dist: trusty
 
+sudo: required
```
Comment (dhensby) [inline .travis.yml]: How come this has been added? What are we using `sudo` for?

Comment (dhensby) [reply .travis.yml]: ok, I've removed this and travis is green

Comment (tractorcow) [reply .travis.yml]: I made many changes until things stopped breaking... this was one of those changes that happened to make the tests pass. :)

File: .gitattributes [hunk 2]
```diff
@@ -1,6 +1,8 @@
-/tests/*.php export-ignore
-/docs export-ignore
+/tests export-ignore
```
Comment (dhensby) [inline .gitattributes]: This breaks the composer autoload classmap for `tests/behat/` as `tests/` doesn't exist.

Comment (tractorcow) [reply .gitattributes]: There are no classes there so we should remove that entire directive.

**Comment by tractorcow** [comment]
I've fixed that in 4.0 branch at https://github.com/silverstripe/silverstripe-siteconfig/pull/86, but we can merge up to 4 branch on merge.

**Pull request merged**